rs/nns/governance/src/neuron/types.rs

```diff
@@ -1447,6 +1484,7 @@ impl From<DecomposedNeuron> for Neuron {
             neuron_type,
             visibility,
             voting_power_refreshed_timestamp_seconds,
+            recent_ballots_next_entry_index: recent_ballots_next_entry_index.map(|x| x as usize),
```

Comment:
it would be more hygienic to use `try_from` instead of `as` and handle the error, but it's unlikely to overflow in practice.

Reply:
Is there any scenario where usize is not at least u32?